cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/schedule/ProgramScheduleMeta.java

```diff
@@ -24,20 +24,18 @@
 public class ProgramScheduleMeta {
 
   private final ProgramScheduleStatus status;
-  private final long lastUpdated;
+  private final long lastUpdateTime;
```

Comment:
is this object stored in a serialized form anywhere or returned as part of some Http endpoint? If so, changing the variable name will be a backwards incompatible change.

It's safer to just change the method name

Reply:
This looks to be used internally and transiently. We convert it to ScheduleDetail that gets returned to client. So the renaming looks to be safe. 

Anyway, I am reverting this renaming change, just to be on the safeside. Will do the renaming in a follow-up change and take a much closer look there.